src/Explorer/Explorer.tsx

```diff
@@ -1394,7 +1394,11 @@ export default class Explorer {
       tab.hashLocation().startsWith(hashLocation)
     ) as TerminalTab[];
 
-    const index = terminalTabs.length + 1;
+    let index = 1;
+    if (terminalTabs.length > 0) {
+      index = terminalTabs[terminalTabs.length - 1].getIndex() + 1;
```

Comment:
Do we know if these are sorted based on index?  Wondering if there will ever be a scenario where the index of the last tab will ever be smaller than the ones before it

Reply:
the new tab always gets added to the end, and will have a higher index than any preceding tab. There will be no case where index of the last tab < index of any other tab.